frontend/src/pages/members/MemberOverviewTab.tsx

```
@@ -0,0 +1,466 @@
+                <div className="space-y-0" style={{ backgroundColor: '#1a1a1a' }}>
+                  {dependants.map((dependant, index) => (
+                    <div key={dependant.id}>
+                      <div className="px-4 py-3 flex items-center justify-between transition-colors border-b"
+                        style={{ backgroundColor: '#1a1a1a', borderBottomColor: '#4a4a4a' }}
+                        onMouseEnter={(e) => {
+                          e.currentTarget.style.backgroundColor = '#2a2a2a';
+                        }}
+                        onMouseLeave={(e) => {
+                          e.currentTarget.style.backgroundColor = '#1a1a1a';
+                        }}
+                      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)